config/initializers/catch_json_parser_errors.rb

```diff
@@ -1,3 +1,3 @@
 unless Rails.env.development?
-  Rails.application.config.middleware.insert_before ActionDispatch::ParamsParser, "CatchJsonParseErrors"
+  Rails.application.config.middleware.use CatchJsonParseErrors
 end
```

Comment:
@kalamkar please remove this middleware `CatchJsonParseErrors`. It causes more problems than helping us.